src/candela/components/Scatter/spec.json

```diff
@@ -45,7 +52,6 @@
           {
             "axis": "y",
             "size": ["@get", "height"],
-            "title": ["@get", "yAxis.title", ["@get", "y", ""]],
```

Comment:
See the diff lines 11-13 above. Instead of putting the default values inline, I just moved them to the top, which I think is better practice and is the point of the `defaults` command.